modules/stochastic_tools/src/transfers/SerializedSolutionTransfer.C

```diff
@@ -169,7 +169,7 @@ SerializedSolutionTransfer::transferInParallel(FEProblemBase & app_problem,
       // Localize the solution and add it to the local container on the rank
```

Comment:
```suggestion
      // Localize the snapshot and add it to the local container on the rank
```

Reply:
Check for similar occurrences!

Reply:
This class still only works for snapshots that are solutions. Because this
checks for variable names. So I think solution is still fine. Later in another
PR, I am creating a `SerializedSnapshotTransfer`, but it will
be general purpose and without variable names.

Reply:
Ah my bad, thanks for the detailed answer!